### ISSUE TITLE
J! 4.3.0 Sorting placehoder

### ISSUE THREAD

**Issue by tonamphong** [gantry/gantry5#3144]
J4.3.0 updated Bootstrap version 5.2 and the placeholder class changed the default Gantry5 sorting style when moving or sorting the grid/particles.

See

![placeholder](https://user-images.githubusercontent.com/2299347/233846886-aaec87e7-c901-431b-9cf4-ee87f24c1001.png)

The placeholder css class located at joomla-root/media/templates/administrator/atum/css/template.min.css

`.placeholder {
    background-color: currentcolor;
    cursor: wait;
    display: inline-block;
    min-height: 1em;
    opacity: .5;
    vertical-align: middle
}`

We can avoid this styling issue by adding the css code to g-admin.css 

`#g5-container .lm-blocks .in-between-grids.placeholder {
    opacity: 1;
    min-height: inherit;
}`

Expected result

![placeholder2](https://user-images.githubusercontent.com/2299347/233847215-e456ed92-c3ce-48cb-a1d9-4fe2dcd28cd0.png)

**Comment by N8Solutions** [comment]
@tonamphong Have you tried the recent CI builds?
https://gantry.org/downloads#ci-builds

If not, please try those and report back.

**Comment by tonamphong** [comment]
> @tonamphong Have you tried the recent CI builds? https://gantry.org/downloads#ci-builds
> 
> If not, please try those and report back.

Sure, latest CI build released: September 29, 2023
The height of placeholder changed and it's hard to drag and drop between the grid block items. You can test the J3 and J4 to see the placeholder's height style between two Joomla version.

**Comment by N8Solutions** [comment]
@tonamphong so am I to understand that you tried the CI build and nothing changed?

**Comment by tonamphong** [comment]
> @tonamphong so am I to understand that you tried the CI build and nothing changed?

YES. Did you notice the different about placeholder between J3 and J4 ?

**Comment by tonamphong** [comment]
@gteigland Can you take a look at this case with Joomla 5? It only affects Joomla; the WordPress and Grav sorting placeholder areas are not affected.

**Comment by monikat12titans** [comment]
@tonamphong This issue has been fixed and included in the latest CI builds. Please download the package from the CI builds section at https://gantry.org/downloads#ci-builds and test it.

**Comment by tonamphong** [comment]
Awesome, thank you
